fractal_server/app/models/job.py

```diff
@@ -115,5 +92,10 @@ class Config:
     end_timestamp: Optional[datetime] = Field(
         default=None, sa_column=Column(DateTime(timezone=True))
     )
-    status: JobStatusType = JobStatusType.SUBMITTED
+    status: str = JobStatusType.SUBMITTED
     log: Optional[str] = None
+
+    @validator("status")
```

Comment:
@mfranzon let's decide whether this validator is actually valuable

Reply:
We can relax the validator as long as there is no user interaction (PATCH) with the job_status

Reply:
Agreed. It doesn't add so much, apart from complexity. Now removed with https://github.com/fractal-analytics-platform/fractal-server/pull/974/commits/83160692fee6a872a411e3c25f0772409ce6ffe8.